src/knot_types.h

```diff
@@ -129,6 +130,7 @@
 #define KNOT_UNIT_ENERGY_CAL				0x05
 #define KNOT_UNIT_ENERGY_KCAL				0x06
 #define KNOT_UNIT_ACCELERATION_MS2			0X01
+#define KNOT_UNIT_ANGULAR_VELOCITY_DEGSEG		0X01
```

Comment:
As the international system of units uses `rad/sec`, shouldn't it be  supported as well?
Not sure which unit is the most common among IMU sensors, but I fell it's not that much of trouble having both of them added now and could save some rework later.

Reply:
I Agree with Paulo.

Reply:
Yeah, it actually should. I've only added deg/seg because it was the unit used by the sensor.
I've updated it with rad/seg as first unit option and deg/seg as second option.